src/screens/User/Profile.js

```diff
@@ -29,7 +29,10 @@ class Profile extends Component {
 	render() {
 		return this.renderContent();
 	}
-
+	/*
+	 * using convertObjectToInitialValues() to pass the value back to values break the app due to parameter
+	 * miss match. Form expected an object but receives an array.
+	 */
```

Comment:
You changed the values to `this.props.activeUser`. This error that you specify in the comment should be fixed in this PR or create a new issue to solve this, depending on the difficulty. I do not understand the need for a comment